### DISCUSSION TITLE
Barre d'amitié avec l'animal de compagnie.

### DISCUSSION THREAD

**Discussion by Kyloros** [Crownicles/Crownicles#670] [Idées acceptées / déjà implémentées]
Juste crée comme une barre de vie pour montré notre niveau d'amitié avec notre animale de compagnie,
pour savoir a quel point ont est proche ou pas.

**Comment by BastLast** [comment]
Pour info on avait pensé à ça en développant les pets et on avait choisi de ne pas le faire afin d'éviter de rendre les familiers trop "robotiques"
Perso je ne serai pas contre l'idée de clarifier l'amour du pet, mais je pense qu’afficher le "score" d'amour n'est pas une bonne solution

**Reply by ghost** [reply]
Et avec des cœurs plus ou moins rempli suivant la relation entre pet et joueur ?

**Reply by Kyloros** [reply]
> Pour info on avait pensé à ça en développant les pets et on avait choisi de ne pas le faire afin d'éviter de rendre les familiers trop "robotiques" Perso je ne serai pas contre l'idée de clarifier l'amour du pet, mais je pense qu’afficher le "score" d'amour n'est pas une bonne solution

a ok

**Comment by BastLast** [comment]
je déplace dans les idées accepté puisque le vétérinaire existe
